tests/unit/test_client.py

```diff
@@ -1626,8 +1626,8 @@ def test_get_signed_policy_v4_virtual_hosted_style(self):
             policy = client.generate_signed_post_policy_v4(
                 BUCKET_NAME,
                 "object-name",
-                conditions=[],
                 expiration=datetime.datetime(2020, 3, 12),
+                conditions=[],
```

Comment:
`conditions=[]` isn't required for this call, I'd prefer removing empty lists in tests to keep them clean.

Reply:
Done